src/xia2/cli/cluster_analysis.py

```diff
@@ -17,64 +16,54 @@
 )
 from dials.util.options import ArgumentParser, reflections_and_experiments_from_files
 from dials.util.version import dials_version
+from dxtbx.model import ExperimentList
 from jinja2 import ChoiceLoader, Environment, PackageLoader
 
 import xia2.Handlers.Streams
-from xia2.Modules.Analysis import batch_phil_scope
-from xia2.Modules.MultiCrystalAnalysis import MultiCrystalAnalysis
+from xia2.Modules.MultiCrystal.cluster_analysis import (
+    cluster_phil_scope,
+    output_cluster,
+    output_hierarchical_clusters,
+)
 from xia2.XIA2Version import Version
 
 logger = logging.getLogger("xia2.cluster_analysis")
 
+xia2_cluster_phil_scope = """\
+clustering
+  .short_caption = "Clustering"
+{
+  output_correlation_cluster_number = None
```

Comment:
We should add a test for these options for cluster_analysis.

Reply:
Have added a test to check for this